dnf/module/repo_module_dict.py

```diff
@@ -70,12 +71,12 @@ def use_default_stream(repo_module):
             if repo_module.conf and \
                     repo_module.conf.locked and \
                     repo_module.conf.version is not None:
-                if repo_module_stream.latest().version != repo_module.conf.version:
-                    logger.info(module_messages[VERSION_LOCKED]
-                                .format("{}:{}".format(repo_module.name, stream),
-                                        repo_module.conf.version))
-
-                repo_module_version = repo_module_stream[repo_module.conf.version]
+                try:
```

Comment:
I'd prefer if-raise than try-raise.
Also add a comment what's being tested and done.